### ISSUE TITLE
Testing version 1.2

### ISSUE THREAD

**Issue by paaljoachim** [munirkamal/gutenberg-forms#94]
Hey 

Here is a feedback on initial testing of version 1.2.
Using:
WordPress 5.4
Twenty Twenty theme
Gutenberg plugin 7.8.1
Chrome.
A local test site using Desktop Server.

Upon adding the Gutenberg Forms block I see this:

<img width="638" alt="Screen Shot 2020-04-11 at 19 36 25" src="https://user-images.githubusercontent.com/5323259/79050641-c17f3a00-7c2b-11ea-846b-6eba1573b4bd.png">

It is nice to see the options Standard - Multi Step or Insert Form.

Selecting Insert form I get a regular form.
I clicked to see what the Styles look like.
Noticed Default is selected and switched to Inline. This is what I see.

The first field Name goes inline. The second field Email does not.
<img width="675" alt="Screen Shot 2020-04-11 at 19 38 24" src="https://user-images.githubusercontent.com/5323259/79050691-0905c600-7c2c-11ea-8a7d-7d019209d5c6.png">

Checked a preview on the frontend, and noticed this:
<img width="634" alt="Screen Shot 2020-04-11 at 19 40 45" src="https://user-images.githubusercontent.com/5323259/79050745-5aae5080-7c2c-11ea-92d0-efd0ed7e2365.png">

Scrolled down the page and added another Gutenberg Forms block.
This time I selected "Insert Form."

Wow.....not sure what to say at initial glanze. This is impressive.

<img width="1345" alt="Screen Shot 2020-04-11 at 19 43 39" src="https://user-images.githubusercontent.com/5323259/79050814-cc869a00-7c2c-11ea-97bf-65c2256e9026.png">

I am scrolling through **Catagories** -> Redo spelling to Categories. (With an e)
That is a lot of forms to choose from. But many of the forms are **very** similar to each other. Just a little different fields and a different title. One will anyway choose something that kinda resembles what one needs and then customize that.

It would be better to start with an option to have 4-5 form types that are very different from each other. 

Some form options:
1. A standard simple contact form.
2. A customer service contact form with contact different departments and each department have some options. Kinda like Technical, Support and Customer service. Having a conditional for each department.
3. Ticket service. Reserve booking/tickets/etc. For instance like this form:
https://spirituellfilm.no/pamelding-til-film/
4. Reserving/ordering items.

Another thing. For instance Online Quiz.
Here one can see the form on the right side. Below the preview of the form could be place to add some info about the form. Such as uses conditional logic and whatever else would be good for the user to know about. 

O I like the Saved forms area. I am looking forward to testing that out!
That would also be an option to import/export from from one site to another.
For instance Caldera Forms have an export/import using Json file.

One thing I miss is having a checkbox some text beside it and a way to link one of the words to another page. For instance the word terms etc is linked to the terms page.

NB. I do not see an insert form button to insert the form I want to take a closer look at.
Ahhh scrolling down I missed the Insert Form button.....

This is impressive. Here is an example of the "Online Order Form" form in the backend.
<img width="687" alt="Screen Shot 2020-04-11 at 19 57 35" src="https://user-images.githubusercontent.com/5323259/79051129-be397d80-7c2e-11ea-91d5-05de1d16c488.png">

Frontend:
<img width="635" alt="Screen Shot 2020-04-11 at 19 58 27" src="https://user-images.githubusercontent.com/5323259/79051143-d3aea780-7c2e-11ea-8b30-738388013e2a.png">

-------------

Sidebar options look good!
I missed having a sidebar option for sending of e-mail to admin.
I went to the toolbar and clicked the mail icon. I began adding fields.

<img width="744" alt="Screen Shot 2020-04-11 at 20 00 50" src="https://user-images.githubusercontent.com/5323259/79051208-3d2eb600-7c2f-11ea-9522-6f53b228d345.png">

Here it would be great to automatically have all the fields included in the Body area. With field title before the field code. So the user can readjust the fields.

Ps. The above is testing on a localhost test site. I will later on test it on a online dev site.

Bottom line! WOW I am impressed! I am looking forward to digging even more into it.

**Comment by ZafarKamal123** [comment]
Let me make this clear, The inline form styling is for each fields inside the form, There is no inline form style for the main gutenberg forms block 🤔
